SourceX/DiabloUI/diabloui.cpp

```diff
@@ -438,11 +348,142 @@ void LoadUiGFX() {
 	LoadArt("ui_art\\heros.pcx", &ArtHero, 4);
 }
 
+// clang-format off
+#define BLANKCOLOR { 0, 0xFF, 0, 0 }
+// clang-format on
+
+void LoadFallbackPalette()

```

Comment:
If this is only ever used in dialogs, perhaps this function should live in `dialogs.cpp`?